### PR TITLE
[FEATURE] Store validation signatures

### PR DESCRIPTION
**Pull request by wilsonianb**
DEC-573

### CODE REVIEW

File: test/models/validation.js [hunk 1]
```diff
@@ -11,15 +11,19 @@ describe('Validation', () => {
...
-      validation_public_key: 'n9LigbVAi4UeTtKGHHTXNcpBXwBPdVKVTjbSkLmgJvTn6qKB8Mqz',
+    const validation = {
+      validation_public_key: 'n9LXZBs2aBiNsgBkhVJJjDX4xA4DoEBLycF6q8zRhXD1Zu3Kwbe4',
       reporter_public_key: 'n9MD5h24qrQqiyBC8aeqqCWvpiBiYQ3jxSr91uiDvmrkyHRdYLUj',
-      ledger_hash: hash
-    })
-    .then(validation => {
-      assert.strictEqual(validation.ledger_hash, hash)
+      ledger_hash: '6B0F79F7447CFAC355748111BB8C816CAE7062FA94675AB30DA237618F3BAD07',
+      signature: '3045022100A762691653A95EEC5B6C820F471482DAF56DB38DA61507889A2E02CEC8CF6C4F02202D08468D83DF8EAC231445382AB21F0046B3516D2A6951FA5C58D54BA16F6492'
```
Comment (stevenzeiler) [inline test/models/validation.js]: Should the `signature` ever be required?

Comment (wilsonianb) [inline test/models/validation.js]: We can't require it if we're still going to use the old log-tailing validation tracker

Comment (stevenzeiler) [inline test/models/validation.js]: good point.

**Comment by stevenzeiler** [comment]
:+1:

**Comment by wilsonianb** [comment]
:+1:

**Pull request merged**